cloud/amazon/ec2_vol.py

```diff
@@ -62,6 +62,11 @@
       - Enable encryption at rest for this volume.
     default: false
     version_added: "1.8"
+  kms_key_id:
+    description:
+      - Specify the id of the KMS key to use.
+    default: null
```

Comment:
Accidentally removed the wrong line. What confused me was, that other parameters also have this line (`default: null`).